exporter/metric/metric_test.go

```diff
@@ -278,19 +239,9 @@ func TestRecordToMpb(t *testing.T) {
 			"foo": "100",
 		},
 	}
-	require.NoError(t, pusher.Stop(ctx))
-
-	aggError := pusher.ForEach(func(library instrumentation.Library, reader export.Reader) error {
-		return reader.ForEach(aggregation.CumulativeTemporalitySelector(), func(r export.Record) error {
-			out := me.recordToMpb(&r, library)
-			if !reflect.DeepEqual(want, out) {
-				return fmt.Errorf("expected: %v, actual: %v", want, out)
-			}
-			return nil
-		})
-	})
-	if aggError != nil {
-		t.Errorf("%v", aggError)
+	out := me.recordToMpb(inputMetrics, inputAttributes, inputLibrary)
```

Comment:
This is now directly invoked with metricdata, rather than through the otel API path.  MeterProvider does not provide a ForEach function anymore (instead has the exporter/reader interfaces), so we can't easily do that anymore. Same for the utf8 version of this test below.